SA3/qa_web/views.py

```diff
@@ -470,5 +470,22 @@ def delete(request, id_):
     return HttpResponseRedirect('/QuestionIndex')
 
 
+# edit Answers
+@login_required(login_url='/login/')
+def editAnswers(request, id_, a_id):
```

Comment:
Can you please change the name of the function to a non-camel case name, such as `edit_answers()`, we want to follow the Python convention and it would be a great help to those working on refactoring the styling throughout the website.